### PR TITLE
Print 'no resource found' in show cmds

### PR DESCRIPTION
**Pull request by maayanf24**
Fix issue https://github.com/submariner-io/enhancements/issues/5

Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

File: pkg/subctl/cmd/show_connections.go [hunk 1]
```diff
@@ -73,9 +73,13 @@ func showConnections(cmd *cobra.Command, args []string) {
 
 func printConnections(connections []connectionStatus) {
 	template := "%-20s%-16s%-16s%-24s%-40s%-16s\n"
-
 	fmt.Printf(template, "GATEWAY", "CLUSTER", "REMOTE IP", "CABLE DRIVER", "SUBNETS", "STATUS")
 
+	if len(connections) == 0 {
+		fmt.Println("no resource found")
```
Comment (skitt) [inline pkg/subctl/cmd/show_connections.go:79]: To match `kubectl`,

```suggestion
		fmt.Println("No resources found.")
```

(and likewise for the other two occurrences).

Comment (maayanf24) [reply pkg/subctl/cmd/show_connections.go:79]: One more question - should the header line be printed in or not in the case of no resources?

Comment (skitt) [reply pkg/subctl/cmd/show_connections.go:79]: > One more question - should the header line be printed in or not in the case of no resources?

`kubectl` doesn’t print the header, so no (see `kubectl get pods -n default` on a newly-deployed cluster for example).

**Comment by skitt** [comment]
@maayanf24 the conflicts need to be resolved manually.

**Pull request merged**